qgisprocess_user_2024.Rmd

```diff
@@ -111,7 +111,7 @@ layout: false
 
 --
 
-- Unified interface to QGIS, GRASS GIS, SAGA, GDAL and other processing providers (1000+ geospatial algorithms)
+- Unified interface to QGIS, GRASS GIS, SAGA, GDAL and other processing providers (1000+ geospatial algorithms from QGIS and plugins)
```

Comment:
This is actually becoming too long. For now I think I'll just revert the change on this line, because the providers are already mentioned. @Robinlovelace let me know if there's some alternative you deem more appropriate.

Reply:
No problem.